### DISCUSSION TITLE
调用返回快捷键

### DISCUSSION THREAD

**Discussion by loikuan** [ywmoyue/biliuwp-lite#110] [feature]
![屏幕截图 2023-05-20 141832](https://github.com/ywmoyue/biliuwp-lite/assets/128059419/805b4a41-2e29-4aa4-bd83-67ee7ead037b)

windows 系统自带uwp应用可以 ait + 左方向键调用返回 ,希望bili lite也支持

**Comment by ywmoyue** [comment]
返回操作是指关闭当前tab吗

**Reply by loikuan** [reply]
我是在单窗口模式下使用的,就只是单纯的返回上一个界面,也就是右上角那个返回按钮的功能

**Reply by loikuan** [reply]
如果是多窗口模式下,这个返回便是回到上一个标签页

**Reply by ywmoyue** [reply]
我没用过单窗口或者多窗口模式，我用一段时间试试

**Reply by loikuan** [reply]
能实现吗?

**Reply by ywmoyue** [reply]
不知道，预计不会优先处理，可能会先处理单窗口模式的显示bug
